themes/nswds/app/frontend/src/scss/components/waratah/_fixes.scss

```diff
@@ -34,3 +34,17 @@
     }
   }
 }
+
+
+
+/**
+ * Add specific class for bitmap icons
+ */
+.nsw-content-block__image {
+  .nsw-content-block__icon {
+    img {
```

Comment:
Can this @extend https://github.com/digitalnsw/nsw-design-system/blob/master/src/components/content-block/_content-block.scss#L52 ?